### ISSUE TITLE
Tags in images

### ISSUE THREAD

**Issue by dcerbolinha** [JoomGallery/JoomGallery#161]
Hello,

I would like to know if there is a plugin under development or a way to add the tags option to the joomgallery images.

Thank you.

**Comment by MrMusic** [comment]
Hi,
I do not know that anyone would have done it.
But it would be nice if someone would do that :-)

**Comment by Elfangor93** [comment]
Yes, adding Tags to the JoomGallery component would be very nice

**Comment by Elfangor93** [comment]
Would it make sense to implement the Tags of the Joomla-Core (https://docs.joomla.org/J3.x:Developing_an_MVC_Component/Adding_Tags) or better create our own Tag-functionality?
As I can see JoomGallery also doesent use categories of the Joomla-Core, maybe it is a strategical thing not to use functions of the Joomla-Core...

Does anyone has experience in adding Tags to a component?

**Comment by MrMusic** [comment]
I would recommend using the tags of the Joomla core and not building a own tag functionality.

There has been an attempt to integrate Joomla-tags into the Joomgallery, see [here:](http://www.joomlaportal.de/module-komponenten-suche-bzw-entwicklung/317605-schlagwort-tags-plugin-f-r-komponente-joom-gallery-gesucht.html)
But it was not finished.
